runtime-common/stdlib/msgpack/adaptors.h

```diff
@@ -28,16 +29,19 @@ struct convert_integer_sign<T, true> {
   static T convert(const msgpack::object& o) {
     if (o.type == stored_type::POSITIVE_INTEGER) {
       if (o.via.u64 > static_cast<uint64_t>(std::numeric_limits<T>::max())) {
-        throw type_error{};
+        RuntimeContext::get().set_msgpack_error(TYPE_ERROR);
+        return T{};
```

Comment:
Can type T be non default constructible?

Reply:
All runtime and codegen classes are default constructible